contrib/localnet/solana/start-solana.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "making an id"
+solana-keygen new -o /root/.config/solana/id.json --no-bip39-passphrase
+
+solana config set --url localhost
+echo "starting solana test validator..."
+solana-test-validator &
+
+sleep 5
+# airdrop to e2e sol account
+solana airdrop 100
+solana airdrop 100 37yGiHAnLvWZUNVwu9esp74YQFqxU1qHCbABkDvRddUQ
+solana program deploy gateway.so
```

Comment:
The deployment should be done in `zetae2e` and the funding should be done in `zetae2e-setup.sh`. I imagine the airdrop rpc works remotely right? You'd need to copy the `solana` binary into the e2e orchestrator container